stack/tools/src/main/java/org/apache/usergrid/tools/Export.java

```diff
@@ -639,6 +639,7 @@ private void fetchDictionaries(File collectionDir, String collectionName, final
 							}
 							
 							jgDictionaries.writeStartObject();
+							jgDictionaries.writeObjectField("Related Entity", et.getUuid());
```

Comment:
Change the tag from Related Entity to Entity since there is no other entity present, unless there is a specific reason to call it Related Entity